README.md

```diff
@@ -182,27 +177,31 @@ Functions may even be composed directly into custom operators:
 
 ```python
 from expression import compose
-from expression.collections import seq
+from expression.collections import seq, Seq
 
+xs = Seq.of(9, 10, 11)
 custom = compose(
     seq.map(lambda x: x * 10),
     seq.filter(lambda x: x > 100),
     seq.fold(lambda s, x: s + x, 0)
 )
-
 ys = custom(xs)
+
+assert ys == 110
 ```
 
-## Fluent and Functional
+### Fluent and Functional
```

Comment:
this puts all the entries in the list of features at the top at the same heading level